spec/adapter/win_32/select_list_spec.rb

```diff
@@ -1,65 +1,65 @@
 require 'spec_helper'
 
-describe "Win32::SelectList", :if => SpecHelper.adapter == :win_32 do
-  let(:window) { RAutomation::Window.new(:title => "MainFormWindow") }
+describe "Win32::SelectList", if: SpecHelper.adapter == :win_32 do
+  let(:window) { RAutomation::Window.new(title: "MainFormWindow") }
   it "#select_list" do
-    expect(window.select_list(:class => /combobox/i, :index => 1)).to exist
+    expect(window.select_list(class: /combobox/i, index: 1)).to exist
 
     RAutomation::Window.wait_timeout = 0.1
-    expect {RAutomation::Window.new(:title => "non-existent-window").
-            select_list(:class => /combobox/i, :index => 1)}.
+    expect {RAutomation::Window.new(title: "non-existent-window").
+            select_list(class: /combobox/i, index: 1)}.
             to raise_exception(RAutomation::UnknownWindowException)
   end
 
   it "#options" do
-    select_list = window.select_list(:class => /combobox/i, :index => 1)
+    select_list = window.select_list(class: /combobox/i, indexv: 1)
```

Comment:
What is `indexv`?

Reply:
And I'm surprised that having this typo here did not make specs fail. Or maybe they did in the next commit - we'll see :)